src/Tabs/Tabs.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { useRef, useState, useEffect } from 'react';
+import { useKeyboardEvent } from './hooks/useKeyboardEvent';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Tab } from './Tab';
+import { Icon } from '../Icon';
+import { ArrowButton, Container, TabsContainer } from './Tabs.styles';
+import { TabsProps } from './types';
+import { faChevronLeft, faChevronRight } from '@fortawesome/free-solid-svg-icons';
+import { getButtonIconSizeStyles } from './utils';
+
+export const Tabs = ({ tabs, onChange, showScrollButtons = true, size = 'medium' }: TabsProps) => {
+  const tabsContainerRef = useRef<HTMLDivElement>(null);
+  const tabRef = useRef<HTMLDivElement>(null);
+  const [direction, setDirection] = useState<'right' | 'left'>('right');
+  const [activeTab, setActiveTab] = useState(0);
+
+  useKeyboardEvent('ArrowRight', () => {
+    if (tabsContainerRef.current) {
+      if (activeTab < tabs.length - 1) {
+        handleTabChange('right', activeTab + 1, false);
+      }
+    }
+  });
+
+  useKeyboardEvent('ArrowLeft', () => {
+    if (tabsContainerRef.current) {
+      if (activeTab > 0) {
+        handleTabChange('left', activeTab - 1, false);
+      }
+    }
+  });
+
+  useEffect(() => {
+    if (tabsContainerRef.current) {
+      const offsetWidth = tabRef.current?.offsetWidth || 0;
+      tabsContainerRef.current.scrollTo({
+        left: activeTab * offsetWidth,
+        behavior: 'smooth',
+      });
+    }
+  }, [activeTab, tabRef, tabsContainerRef]);
+
+  const handleTabChange = (direction: 'right' | 'left', newActiveTab: number, disabled: boolean) => {
+    if (disabled === false) {
+      setDirection(direction);
+      onChange(newActiveTab);
+      setActiveTab(newActiveTab);
+    }
+  };
+
+  return (
+    <Container display="flex">
```

Comment:
I don't know exactly where, but we can use the proper role for this component accessibility `role: 'tabpanel'`

Reference: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tabpanel_role